openapi-client-native/src/main/java/io/ballerina/openapi/client/GeneratedClient.java

```diff
@@ -80,14 +64,12 @@ private static Object invokeClientMethod(Environment env, BObject client, BArray
         int pathLength = (int) path.getLength();
         int paramLength = (int) params.getLength();
 
-        Object[] paramFeed = new Object[(pathLength + paramLength) * 2];
+        Object[] paramFeed = new Object[pathLength + paramLength];
         for (int i = 0; i < pathLength; i++) {
-            paramFeed[i * 2] = path.get(i);
-            paramFeed[i * 2 + 1] = true;
```

Comment:
May I know why we removed the assignment of `true` at `paramFeed[i * 2 + 1]`?

Reply:
This is a change in the Runtime API. Previously we used `true` or `false` to indicate whether arguments have default values, we don't have to do that anymore.